frameworks/Java/vertx/src/main/java/vertx/WebServer.java

```diff
@@ -28,15 +26,11 @@
 	private static final CharSequence RESPONSE_TYPE_PLAIN = new AsciiString("text/plain");
 	private static final CharSequence RESPONSE_TYPE_JSON = new AsciiString("application/json");
 
-	private static final CharSequence TEXT_MESSAGE = new AsciiString("message");
-	private static final byte[] HELLO_WORLD = "Hello, world!".getBytes();
```

Comment:
@whitingjr IMO This code is acceptable for the `plaintext` test - using a byte array with the values of the plaintext response and just returning it. It is used by other participants. The rules are **different** for it.
I thought that I was clear that only the JSON instantiation is the problem. Also @nbrady-techempower  commented the particular lines that are problematic (#90-91).